awscli/customizations/s3/results.py

```diff
@@ -163,8 +164,11 @@ def __init__(self):
         self.expected_files_transferred = 0
 
         # Keys: result from _get_progress_cache_key()
-        # Values: number of bytes left for a particular transfer
-        self._progress_remaining_cache = {}
+        # Values: number of bytes that have been transferred for that key
+        self._progress_cache = defaultdict(int)
+        # Keys: result from _get_progress_cache_key()
```

Comment:
Yeah I can remove these. I can also look to rename the name of variable/methods. How about?
``_progress_cache`` --> ``_ongoing_progress``
``_total_size_cache`` --> ``_ongoing_total_sizes``
 ``_get_cache_key()`` --> ``_get_result_key()``